clientd3d/config.c

```diff
@@ -102,6 +102,11 @@ static char INIDebug[]        = "Debug";
 static char INISecurity[]     = "Security";
 static char INITechnical[]    = "Technical";
 
+static char config_ini_file[] = "./config.ini";
```

Comment:
You definitely do not want to hard-code and redefine the location of this file, which is already available as ini_file.

Reply:
I'd like you to research this some more and test it in both Steam and Web installer configurations (they run in different directories).  Notice how the main config file has its directory set explicitly:

https://github.com/Meridian59/Meridian59/blob/master/clientd3d/config.c#L172

So I'm not convinced that hard-coding the path to the file will work in all cases.

Also, given that this situation with multiple config files is confusing, you need to document it.